root/static/styles/release-editor.less

```diff
@@ -151,7 +151,6 @@ fieldset.advanced-medium {
             height: 2em;
             border: 0;
             padding: 0 5px;
-            &.preview { background-color: @input-preview-bg; }
```

Comment:
This is unnecessary; these inputs are already matched by the rule in `forms.less`.